src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -141,12 +141,15 @@ void SotaUptaneClient::PackageInstallSetResult(const Uptane::Target &target) {
     }
   }
   storage->storeInstallationResult(result);
+  return result;
 }
 
 void SotaUptaneClient::reportHwInfo() {
   Json::Value hw_info = Utils::getHardwareInfo();
   if (!hw_info.empty()) {
     http->put(config.tls.server + "/core/system_info", hw_info);
+  } else {
+    LOG_WARNING << "Unable to fetch hardware information from host system.";
```

Comment:
Cool!

Reply:
Yeah, that probably should've been there ages ago.